### ISSUE TITLE
Regarding some issues with the "plate_info.txt" in the dataset.

### ISSUE THREAD

**Issue by YeahTOT** [haoyGONG/LPDGAN#2]
Hello, I am very interested in your research, but I encountered some issues when trying to replicate it. In the dataset you provided, I did not find the "**plate_info.txt**" file. I would like to know what the format of "plate_info.txt" should be? From the source code, it seems that it should contain the names of the images and their corresponding labels.

**Comment by haoyGONG** [comment]
Thank you for your interest in this project. For the "plate_info.txt" file, we are in the process of verifying its contents and will make the file public once we have completed this work. In addition, if you have a tool to automatically recognize license plates, you can use sharp license plate images to generate such a label txt file. An example of the format of this file is:
```
grab10.jpg,苏UZ502G
grab10231.jpg,苏E56V3X
grab10238.jpg,苏EJ8B72
```
Before reading the model, please convert the text license plate number into vector form. The vector length in this project is 21, whcih can be changed by modifying the code in models/swin_transformer.py, line 1149 and 1152. The comparison table is as follows:
`"#京沪津渝冀晋蒙辽吉黑苏浙皖闽赣鲁豫鄂湘粤桂琼川贵云藏陕甘青宁新学警港澳挂使领民航危0123456789ABCDEFGHJKLMNPQRSTUVWXYZ险品"`

**Comment by haoyGONG** [comment]
Thank you for your interest in this project. For the "plate_info.txt" file, we are in the process of verifying its contents and will make the file public once we have completed this work. In addition, if you have a tool to automatically recognize license plates, you can use sharp license plate images to generate such a label txt file. An example of the format of this file is:
```
grab10.jpg,苏UZ502G
grab10231.jpg,苏E56V3X
grab10238.jpg,苏EJ8B72
```
Before reading the model, please convert the text license plate number into vector form. The vector length in this project is 21, whcih can be changed by modifying the code in models/swin_transformer.py, line 1149 and 1152. The comparison table is as follows:
`"#京沪津渝冀晋蒙辽吉黑苏浙皖闽赣鲁豫鄂湘粤桂琼川贵云藏陕甘青宁新学警港澳挂使领民航危0123456789ABCDEFGHJKLMNPQRSTUVWXYZ险品"`

**Comment by haoyGONG** [comment]
Thank you for your interest in this project. For the "plate_info.txt" file, we are in the process of verifying its contents and will make the file public once we have completed this work. In addition, if you have a tool to automatically recognize license plates, you can use sharp license plate images in LPBlur to generate such a label txt file. An example of the format of this file is:
```
grab10.jpg,苏UZ502G
grab10231.jpg,苏E56V3X
grab10238.jpg,苏EJ8B72
```
Before passing in the model, please convert the text license plate number into vector form. The vector length in this project is 21, whcih can be changed by modifying the code in `models/swin_transformer.py, line 1149 and 1152`. The comparison table is as follows:
`"#京沪津渝冀晋蒙辽吉黑苏浙皖闽赣鲁豫鄂湘粤桂琼川贵云藏陕甘青宁新学警港澳挂使领民航危0123456789ABCDEFGHJKLMNPQRSTUVWXYZ险品"`

**Comment by wucod** [comment]
May I ask when this document can be released?

**Comment by NUAA-XSF** [comment]
I would like to ask when the plate_info.txt file will be uploaded. Thank you very much

**Comment by Tsai-chia-hsiang** [comment]
I encounter the problem when executing these two lines from https://github.com/haoyGONG/LPDGAN/blob/main/data/LPBlur_dataset.py : 
```
plate_info = np.fromstring(plate_info, sep=' ')
```
``` 
plate_info = torch.from_numpy(plate_info)
```

```
plate_info MJW-6302 tensor([], dtype=torch.float64)
plate_info BJB-6317 tensor([], dtype=torch.float64)
plate_info NHR-1552 tensor([], dtype=torch.float64)
```
which cause a very small number by ```np.fromstring()``` and hence trigger the error when dataloader stacing the tensors.

Could you please show the process of embedding the plate_info into a vector for this project in order to train ? 
Thanks

**Comment by NUAA-XSF** [comment]
> I encounter the problem when executing these two lines from https://github.com/haoyGONG/LPDGAN/blob/main/data/LPBlur_dataset.py :
> 
> ```
> plate_info = np.fromstring(plate_info, sep=' ')
> ```
> 
> ```
> plate_info = torch.from_numpy(plate_info)
> ```
> 
> ```
> plate_info MJW-6302 tensor([], dtype=torch.float64)
> plate_info BJB-6317 tensor([], dtype=torch.float64)
> plate_info NHR-1552 tensor([], dtype=torch.float64)
> ```
> 
> which cause a very small number by `np.fromstring()` and hence trigger the error when dataloader stacing the tensors.
> 
> Could you please show the process of embedding the plate_info into a vector for this project in order to train ? Thanks

The code implementation here is different from that in the paper. According to the approach in the paper, the sharp image should be input into the pre-trained CRNN to obtain the vector, and then the loss function should be calculated.
